main.js

```diff
@@ -1,41 +1,39 @@
-
-// Adds event listener to trigger everytime there is a keypress. It then passes that keypress into the 'move' function.
-window.addEventListener('keydown',
-    function(event) {
-        if (lives > 0) {
-        var keypress = event.keyCode;
-        move(keypress);
-        }
-    })
-
 // Initialize canvas element
 var game = document.getElementById('game');
 var ctx = game.getContext('2d');
 
 // Include sprites
 var sprites = new Image();
-sprites.src = 'assets/frogger-sprites.png';
 var deathSprite = new Image();
-deathSprite.src = 'assets/skull-sprite.png'
 var gameOverSprite = new Image();
+
+sprites.src = 'assets/frogger-sprites.png';
+deathSprite.src = 'assets/skull-sprite.png'
 gameOverSprite.src = 'assets/gameOverSprite.png';
 
 // Set score variables
+// currentScore is used to ensure you only gain points when you reach a
+// new max row, it's a running total at any point and can decrement.
 var score = 0;
+var currentScore = 0;
 var highScore = 0;
+if (window.localStorage['highScore']) {
+    highScore = localStorage['highScore'];
+} 
+
 var lives = 3;
-    // Variable to hold current high score to display as 'score'. This way the
-    // player cannot score by going backward and forward over and over in the
-    // same place.
-var currentScore = 0;
+
+// Original Frog Size from Sprite Sheet
+var frogWidth = 30;
+var frogHeight = 22;
 
 // Frog position
-var posX = 200;
-var posY = 530;
+var frogX = 200;
+var frogY = 530;
 var facing = 'up';
 
-// Time variable
-var sec = 10;
+// Sets game speed by rendering a frame every xx milliseconds
+var gameSpeed = 30;
```

Comment:
I don't think we need to change it for this pull request but it would be nice if the gameSpeed was set closer to where the map is initialized. Just for better organization.